test/test_unary_ufuncs.py

```diff
@@ -1266,11 +1266,30 @@ def check_equal(t):
             self.assertEqual(actual, expected)
 
         range = (-10, 10)
+        t = torch.linspace(*range, 1, device=device, dtype=dtype)
+        check_equal(t)
 
-        t = torch.linspace(*range, int(1e4), device=device, dtype=dtype)
+        # Skip testing NaN, inf, -inf since they are tested in reference_numerics tests.
+        info = torch.finfo(dtype)
+        min, max, eps, tiny = info.min, info.max, info.eps, info.tiny
+        t = torch.tensor([min, max, eps, tiny], dtype=dtype, device=device)
+        check_equal(t)
+    
+    @dtypes(torch.float32, torch.float64)
+    @unittest.skipIf(not TEST_SCIPY, "SciPy not found")
+    def test_special_log_ndtr_vs_scipy(self, device, dtype):
+        def check_equal(t):
+            # Test by comparing with scipy
+            actual = torch.special.log_ndtr(t)
+            expected = scipy.special.log_ndtr(t.cpu().numpy())
+            self.assertEqual(actual, expected)
+        
+        # covering special implementation cases for values < -20 or > 6
+        range = (-30, 10)  
+        t = torch.linspace(*range, 1, device=device, dtype=dtype)
         check_equal(t)
 
-        # NaN, inf, -inf are tested in reference_numerics tests.
+        # Skip testing NaN, inf, -inf since they are tested in reference_numerics tests.
         info = torch.finfo(dtype)
         min, max, eps, tiny = info.min, info.max, info.eps, info.tiny
```

Comment:
Should these test cases be included in the OpInfo tests?

Reply:
can't think of a reason not to

cc. @kshitij12345